app/models/manageiq/providers/lenovo/manager_mixin.rb

```diff
@@ -88,7 +88,8 @@ def discover(ip_address, port)
         new_ems = create!(
           :name     => "Discovered Provider ##{count + 1}",
           :hostname => URI('https://' + ip_address),
```

Comment:
I know it isn't a change made by your PR, but could you change this line to use `URI::HTTPS.build`?

Reply:
Actually we don't require the protocol prefix at all any more, I think we should remove the 'https://' altogether and just pass in the ip_address into this method.

Reply:
The above change can be a separate PR though.   This LGTM.

Reply:
Sounds good for me